docs/index.html

```diff
@@ -505,27 +505,27 @@ <h4 id="a-micro-micro-framework">~ a micro micro-framework ~</h4>
 <h2 id="hello-world">Hello World</h2>
 <p>Here’s the bare minimum:</p>
 <div class="sourceCode" id="cb1"><pre class="sourceCode rust"><code class="sourceCode rust"><span id="cb1-1"><a href="#cb1-1"></a><span class="pp">vial::routes!</span> <span class="op">{</span></span>
-<span id="cb1-2"><a href="#cb1-2"></a>    GET <span class="st">&quot;/&quot;</span> =&gt; |_| <span class="st">&quot;Greetings, creature.&quot;</span>;</span>
+<span id="cb1-2"><a href="#cb1-2"></a>    GET <span class="st">&quot;/&quot;</span> <span class="op">=&gt;</span> <span class="op">|</span>_<span class="op">|</span> <span class="st">&quot;Greetings, creature.&quot;</span><span class="op">;</span></span>
```

Comment:
Most of this diff is from rendering with a different version of pandoc.

Reply:
No problem, I might just make the docs generation into a GH:Action so we don't have to play wack-a-mole with non-cargo dev dependency versioning. Either that or modify it to use syntect + pulldown-cmark so everyone *is* on the same version. I'm not sure that "just install whatever and then run this random Ruby in the Makefile" is setting future generations of contributors up for success...